main.py

```diff
@@ -3280,10 +3288,16 @@ def update_weather_data(
             logger.error(f'获取天气数据出错：{weather_data}')
             try:
                 if hasattr(self, 'weather_icon'):
-                    self.weather_icon.setIcon(QIcon(CW_HOME / 'img/weather/99.svg'))
+                    self.weather_icon.setIcon(QIcon(str(CW_HOME) + '/img/weather/99.svg'))

```

Comment:
你可以使用 str() ，但是更为优雅的应该是

```suggestion
                    self.weather_icon.setIcon(QIcon(f"{CW_HOME / 'img/weather/99.svg'}"))

```

而不是拼接字符串。